sessionctx/variable/tidb_vars.go

```diff
@@ -399,6 +399,9 @@ const (
 
 	// TiDBEnableClusteredIndex indicates if clustered index feature is enabled.
 	TiDBEnableClusteredIndex = "tidb_enable_clustered_index"
+
+	// TiDBSlowLogMasking indicates that whether masking the query data when log slow query.
```

Comment:
Does it only effective for parameterized queries?

Reply:
What's your mean of `parameterized queries`?

Actually the masking SQL is also used to generate the SQL digest.

Reply:
Oh, I mean prepared statements. For non prepared statements, will they have effect?

Reply:
yes, It also works on non-prepared statement.